INSTALL.md

```diff
@@ -100,4 +104,7 @@ Avoid using the `*= require_self`, `*= require_tree .`, and `*= require` stateme
   @import "nypl-normalize";
   ```
 
-**Important:** Make sure the file has `.scss` extension (or `.sass` for Sass syntax).
\ No newline at end of file
+**Important:** Make sure the file has `.scss` extension (or `.sass` for Sass syntax).
+
+## <a name="pa11y"></a>Accessiblity Testing with pa11y
```

Comment:
You should add that you need to start a server at localhost:4000 before running the tests.

Reply:
And that you need to run `npm run test-pa11y`.

Reply:
I'll add instructions to the install docs...